### PR TITLE
Dev/quoting v4

### PR DESCRIPTION
**Pull request by JerrolKrause**

### CODE REVIEW

File: apps/customer/src/app/routes/quote/quote.component.ts [hunk 1]
```diff
@@ -38,13 +37,22 @@ export class QuoteComponent implements OnInit {
...
-    private socket: SocketService,
     private quoteSvc: QuotingService,
     private storage: AppStorageService
   ) {}
 
   ngOnInit(): void {
+    this.sub = this.storage.quoteActive$
+      .pipe(filter((x) => !!x))
+      .subscribe((quoteActive) => {
+        console.log(quoteActive);
```
Comment (archanglmr) [inline apps/customer/src/app/routes/quote/quote.component.ts]: Do you want the console.log still?

File: apps/customer/src/app/routes/quote/quote.component.ts [hunk 2]
```diff
@@ -38,13 +37,22 @@ export class QuoteComponent implements OnInit {
 
   public isUpdating = false;
 
+  private sub: Subscription | null = null;
```
Comment (archanglmr) [inline apps/customer/src/app/routes/quote/quote.component.ts]: If you always do `subscriptions = new Subscription()` you can do `this.subscriptions.add(x.subscribe())` and in destroy `this.subscriptions.unsubscribe()` and never need to do a type with `| null` or any extra checking.

File: apps/loan/loan-products/src/app/components/loan-products-grid/loan-products-grid.component.html [hunk 3]
```diff
@@ -36,7 +36,11 @@ <h2 class="mb-0">
       let-expanded="expanded"
       let-rowIndex="rowIndex"
     >
-      <tr>
+      <tr
+        [ngClass]="{
```
Comment (archanglmr) [inline apps/loan/loan-products/src/app/components/loan-products-grid/loan-products-grid.component.html:40]: FYI: This is a fine technique. I prefer `[class.row-selected]="quoteSelected() === rowIndex"` so the classes don't grow into a big dynamic JSON object.

File: apps/loan/loan-products/src/app/loan-products.component.ts [hunk 4]
```diff
@@ -42,7 +42,12 @@ export class LoanProductsComponent {
     public dialogService: DialogService,
     public storage: AppStorageService,
     private route: ActivatedRoute
-  ) {}
+  ) {
+    /***/
+    this.lpSvc.plaidStore.state$.subscribe((state) =>
+      console.warn(state.data?.plaid?.link)
```
Comment (archanglmr) [inline apps/loan/loan-products/src/app/loan-products.component.ts:48]: Still want the `console.log()`?

File: apps/loan/loan-products/src/app/shared/services/loan-products.service.ts [hunk 5]
```diff
@@ -37,7 +47,33 @@ export class LoanProductsService {
...
+    autoLoad: false,
+    getQuery: LinkDocument,
+  });
+
+  constructor(
+    private graphSvc: GraphQLStoreCreatorService,
+    public dialogService: DialogService,
+    public storage: AppStorageService
+  ) {
+    this.pingStore.getData().subscribe();
```
Comment (archanglmr) [inline apps/loan/loan-products/src/app/shared/services/loan-products.service.ts:65]: Without having looked to hard, does this need to unsubscribe somewhere?

**Review by archanglmr — APPROVED** [review]
Adjustments are optional.

File: apps/customer/src/app/routes/quote/quote.component.ts (hunk 1 above)
Comment (JerrolKrause) [reply apps/customer/src/app/routes/quote/quote.component.ts]: Thx, removed

File: apps/loan/loan-products/src/app/loan-products.component.ts (hunk 4 above)
Comment (JerrolKrause) [reply apps/loan/loan-products/src/app/loan-products.component.ts:48]: Yeah, needed to confirm the graph link is working

File: apps/loan/loan-products/src/app/shared/services/loan-products.service.ts (hunk 5 above)
Comment (JerrolKrause) [reply apps/loan/loan-products/src/app/shared/services/loan-products.service.ts:65]: It's a cold observable like an http call.

**Pull request merged**